packages/client/hmi-client/src/components/model/petrinet/tera-model-description.vue

```diff
@@ -102,24 +143,27 @@ import TeraModelDiagram from '@/components/model/petrinet/model-diagrams/tera-mo
 import TeraModelEquation from '@/components/model/petrinet/tera-model-equation.vue';
 import TeraModelObservable from '@/components/model/petrinet/tera-model-observable.vue';
 import { isDataset, isDocument, isModel } from '@/utils/data-util';
-import TeraColumnarPanel from '@/components/widgets/tera-columnar-panel.vue';
 import Column from 'primevue/column';
 import DataTable from 'primevue/datatable';
-import TeraModelCard from './tera-model-card.vue';
 import TeraModelSemanticTables from './tera-model-semantic-tables.vue';
 
 const props = defineProps<{
 	model: Model;
-	modelConfigurations: ModelConfiguration[];
-	highlight: string;
-	featureConfig: FeatureConfig;
+	modelConfigurations?: ModelConfiguration[];
+	highlight?: string;
+	featureConfig?: FeatureConfig;
 }>();
 
 const emit = defineEmits(['update-model', 'fetch-model', 'update-configuration', 'model-updated']);
 
 const teraModelDiagramRef = ref();
 
 const card = computed(() => {
+	// prioritize gollm_card over skema card
+	if (props.model.metadata?.gollmCard) {
+		return props.model.metadata.gollmCard;
+	}
```

Comment:
I do not think we should do that. This depends on the TA1-TA4 service toggle.
Let me ask @pascaleproulx about it and come back to it.

Reply:
Let me know...I guess in the model view there should be a toggle for ta1/ta4 to see which should be shown.  There may be instances where a model is enriched both by skema and gollm, I just made a call and prioritized ta4 over ta1 in that case

Reply:
Let's merge this and make a new PR once we have an answer for it.